src/amo/components/AddonMeta/index.js

```diff
@@ -30,7 +31,14 @@ export class AddonMetaBase extends React.Component<Props> {
       userCount = null;
       userTitle = i18n.gettext('Users');
     } else if (averageDailyUsers) {
-      userCount = i18n.formatNumber(averageDailyUsers);
+      userCount = addon && addon.public_stats ? (
```

Comment:
I removed the stats from the user count in this part of the page; there is a link to the statistics dashboard in the `More Info` card, that's where this link should be added: https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonMoreInfo/index.js#L60